client/src/components/product_detail_page.jsx

```diff
@@ -1,25 +1,49 @@
 import React from 'react';
+import Axios from 'axios';
```

Comment:
don't know if we should keep it lowercase but that's how I've always seen it done